docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,98 @@
+# Running Ollama on AMD iGPU 780M
+
+Ollama could run on the iGPU 780M of AMD Ryzen CPU at Linux bases on ROCm. There only has a little extra settings than Radeon dGPU like RX7000 series.
+
+## Keys for usage
+- Ryzen 7000s/8000s CPU with iGPU 780M
+- amdgpu driver and rocm6.0
+- Linux OS is required (Windows and WSL2 are not supported)
+- BIOS must be set to enable the iGPU and dedicate > 1GB RAM to VRAM
```

Comment:
The user might also need headroom to run their desktop environment. It's worth mentioning that the user will need the size of their model + around 1-2GBs for desktop and GUI overhead + model context. Because llamacpp doesn't support UMA yet, the user will probably have a bad time with anything under 6gigs. Realistically you'd want 8 or 16. The following blog has some good reasoning on this. https://blog.machinezoo.com/Running_Ollama_on_AMD_iGPU

Reply:
We have "0 Set UMA for iGPU in BIOS. (at least >1GB, recommend to >8GB for Llama3:8b q4_0 model size is 4.7GB)" in Prerequisites.